doc/05-service-monitoring.md

```diff
@@ -651,7 +651,7 @@ Icinga sets `LC_NUMERIC=C` to enforce this locale on plugin execution.
 ```
 
 The UoMs are written as-is into the [core backends](14-features.md#core-backends)
-(Icinga DB, IDO, API). I.e. 12.445000ms remain 12.445000ms.
+(IDO, API). I.e. 12.445000ms remain 12.445000ms.
```

Comment:
Now no longer affects all core backends, given that Icinga DB is out. Also, would it make sense to expose the normalized perfdata over the API?

Reply:
Once someone needs it...